.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "metrics/test-data"]
+	path = metrics/test-data
```

Comment:
Please put this under a third_party/maxmind directory.

Reply:
Done